modules/backend/widgets/Table.php

```diff
@@ -65,6 +66,16 @@ public function init()
 
         $this->recordsKeyFrom = $this->getConfig('keyFrom', 'id');
 
+        // Validate limits
+        if (!empty($this->getConfig('maxItems')) && (int) $this->getConfig('minItems', 1) > (int) $this->getConfig('maxItems')) {
+            throw new ApplicationException(
```

Comment:
This should be a SystemException, ApplicationExceptions are not logged and are meant as validation for the user. In this case the user doesn't control this configuration - the developer does.

Reply:
Good point - changed.